socialhome/streams/views.py

```diff
@@ -48,14 +48,11 @@ def dump_content(content):
                 "timestamp": content.formatted_timestamp,
                 "humanizedTimestamp": content.humanized_timestamp,
                 "edited": bool(content.edited),
-                "childrenCount": content.reply_count,
+                "repliesCount": content.children_count,
```

Comment:
It's now `content.reply_count` in master 👍

Reply:
Breaking change are evil! :D